extensions/spark/kyuubi-spark-authz/src/main/resources/table_command_spec.json

```diff
@@ -582,6 +582,24 @@
     "fieldName" : "query",
     "fieldExtractor" : "LogicalPlanQueryExtractor"
   } ]
+},{
+  "classname" : "org.apache.spark.sql.catalyst.plans.logical.ReplaceData",
+  "tableDescs" : [ {
+    "fieldName" : "table",
+    "fieldExtractor" : "DataSourceV2RelationTableExtractor",
+    "columnDesc" : null,
+    "actionTypeDesc" : {
+      "fieldName" : null,
+      "fieldExtractor" : null,
+      "actionType" : "UPDATE"
+    },
+    "tableTypeDesc" : null,
+    "catalogDesc" : null,
+    "isInput" : false,
+    "setCurrentDatabaseIfMissing" : false
+  } ],
+  "opType" : "UPDATE",
```

Comment:
This can be `QUERY` as we have already defined `actionType`.

Reply:
+1, and skip adding UPDATE to OperationType.

Reply:
> This can be `QUERY` as we have already defined `actionType`.

I tried to change it to query, but the update permission is still uncontrollable

Reply:
If what you said is true, this could be a serious bug, the `actionType` shall be respected first.

Reply:
> If what you said is true, this could be a serious bug, the `actionType` shall be respected first.

<img width="1531" alt="image" src="https://user-images.githubusercontent.com/33471639/223965055-fca12eba-f6a7-414a-bc14-5e86cecbaa0f.png">

![image](https://user-images.githubusercontent.com/33471639/223965332-47b745b0-0243-4065-8175-7859f8300c5d.png)

Reply:
> This can be `QUERY` as we have already defined `actionType`.

When I change to query, 
<img width="914" alt="image" src="https://user-images.githubusercontent.com/33471639/223972691-e5b6aa0e-3c8b-4a78-ae11-fdc4d23ebda3.png">

I get the following error：
<img width="1544" alt="image" src="https://user-images.githubusercontent.com/33471639/223972898-703a7bc1-8474-422f-8aaa-706fc8027151.png">

Reply:
It seems that `inputs ===> ArrayBuffer....., OTHER` as the `actionType` is wrong, which should be UPDATE as you added in the json config

Reply:
I know where the bug is,
In org.apache.kyuubi.plugin.spark.authz.PrivilegesBuilder#build did not match the new rule.

Consider add 

```diff
PrivilegesBuilder.scala
@@ -220,6 +220,8 @@ object PrivilegesBuilder {
     val inputObjs = new ArrayBuffer[PrivilegeObject]
     val outputObjs = new ArrayBuffer[PrivilegeObject]
     val opType = plan match {
+      case c if isKnownTableCommand(c) =>
+        buildCommand(c, inputObjs, outputObjs, spark)
       // RunnableCommand
       case cmd: Command => buildCommand(cmd, inputObjs, outputObjs, spark)
       // Queries

```

Reply:
This line is necessary. Command trait may not cover all cases for buildCommand